tests/integration/test_analysisd/test_eps/data/configuration_template/configuration_enabled.yaml

```diff
@@ -0,0 +1,49 @@
+- sections:
+    - section: global
+      elements:
+        - limits:
+            elements:
+              - eps:
+                  elements:
+                    - maximum:
+                        value: MAXIMUM
+                    - timeframe:
+                        value: TIMEFRAME
+
+    - section: remote
+      elements:
+        - connection:
+            value: secure
+        - port:
+            value: '1514'
+        - protocol:
+            value: TCP
+
+    - section: sca
+      elements:
+        - enabled:
+            value: 'no'
+
+    - section: rootcheck
+      elements:
+        - disabled:
+            value: 'yes'
```

Comment:
Why enable this?

Reply:
Disabled modules

Reply:
This should be disabled by a configuration library